suricata.yaml.in

```diff
@@ -193,6 +193,7 @@ outputs:
         - dns:
             # control logging of queries and answers
             # default yes, no to disable
+            # more compact log formats when using "split" or "unified"
             query: yes     # enable logging of DNS queries
             answer: yes    # enable logging of DNS answers
             # control which RR types are logged
```

Comment:
I can't help but feel like there are too many states this can be in.  I know the filtering is already in, but I wonder if we can still do some changes to make it simpler:

```
- dns:
    # The style of logging:
    #    discrete: the classic style of an event per question and answer
    #    split: one event per request, one event per response
    #    unified: one event containing request and response
    style: split
    # Types of RR types to log, by default all are logged.
    #types: [a, aaaa, cname, mx, ns, ptr, txt]
```

Right now I think its a bit confusing.  Would you want query and answer to have different values?  And custom appears to be a filter when you have "yes" and "yes", but it doesn't apply for split and unified?  Might be nice to have it apply to all modes?